content/docs/start/experiments/building-pipelines.md

```diff
@@ -71,6 +67,8 @@ stages:
       - data/test_data
 ```
 
+`dvc exp run` will run all stages in the `dvc.yaml` file.
```

Comment:
Not blocking: 

Do you think is worth adding a ```console block showing a simplified output (like the `Running stage `data_split`...)

Commenting mostly because of the paragraph bellow where you introduce the skip/cache